website/docs/docs/dbt-cloud/on-premises/prerequisites.md

```diff
@@ -39,7 +39,40 @@ Customers hosting dbt Cloud on Microsoft Azure should use [Azure Database for Po
 
 dbt Cloud supports S3 (and S3-compatible APIs) or Azure Blob Storage as its object storage solution for logs and run artifacts. You will need to provide two buckets (s3) or containers (azure) to dbt Cloud: one for logs generated by your dbt runs, and another for all of the other artifacts generated by your dbt runs.
 
-On AWS, you can use either instance profile based or AWS keypair based authentication with S3. The instance profile or keypair should be provisioned before creating the instance and installing the application.
+On AWS, you can use either instance profile based or AWS keypair based authentication with S3. The instance profile or keypair should be provisioned with the below IAM permissions before creating the instance and installing the application.
+
+
+```json
+{  
+    "Version": "2012-10-17",  
+    "Statement": [  
+        {  
+            "Sid": "VisualEditor0",  
+            "Effect": "Allow",  
+            "Action": [  
+                "s3:PutObject",  
+                "s3:GetObjectAcl",  
+                "s3:GetObject",  
+                "s3:ListBucket"  
+            ],  
+            "Resource": [  
+                "arn:aws:s3:::<bucket-name>-logs",
+                "arn:aws:s3:::<bucket-name>-logs/*",   
+                "arn:aws:s3:::<bucket-name>-artifacts",  
+                "arn:aws:s3:::<bucket-name>-artifacts/*"  
+                 
+            ]  
+        },  
+        {  
+            "Sid": "VisualEditor1",  
+            "Effect": "Allow",  
+            "Action": "s3:HeadBucket",  
+            "Resource": "*"  
+        }  
```

Comment:
once you move headbucket up, you should be able to drop this block entirely

Reply:
removed